s2cloudless/PixelClassifier.py

```diff
@@ -88,9 +88,12 @@ def image_predict_proba(self, X):
         :return: classification probability map
         :rtype: numpy array, [n_samples, n_pixels_y, n_pixels_x]
         """
-
         pixels = self.extract_pixels(X)
 
-        probabilities = self.classifier.predict_proba(pixels)
+        if hasattr(self.classifier, 'predict_proba'):
+            probabilities = self.classifier.predict_proba(pixels)
+        else:
+            probabilities = self.classifier.predict(pixels)
+            probabilities = np.vstack((1. - probabilities, probabilities)).transpose()
```

Comment:
I don't understand this. Which classifier does this?

Reply:
In previous versions, when we used pickled model, we had here an instance of `lightgbm.LGBMClassifier` class. It had methods `predict` and `predict_proba`. Now the model is loaded in `lightgbm.Booster` class and it has only method `predict`, which works differently than `LGBMClassifier.predict`.

I have just got an idea of how to write this in a much more explicit way -> will change.

Reply:
Change added